tools/server/server.cpp

```diff
@@ -4959,7 +4961,14 @@ int main(int argc, char ** argv) {
     SetConsoleCtrlHandler(reinterpret_cast<PHANDLER_ROUTINE>(console_ctrl_handler), true);
 #endif
 
-    LOG_INF("%s: server is listening on http://%s:%d - starting the main loop\n", __func__, params.hostname.c_str(), params.port);
+    std::string full_url;
+    if (is_sock) {
+        full_url = params.hostname;
+    } else {
+        full_url = "http://" + params.hostname + ":" + std::to_string(params.port);
+    }
+
+    LOG_INF("%s: server is listening on %s - starting the main loop\n", __func__, full_url.c_str());
```

Comment:
nits: I prefer the log code to be short, so something like this looks better:

```cpp
    LOG_INF("%s: server is listening on %s - starting the main loop\n", __func__,
            is_sock ? params.hostname.c_str() : string_format("http://%s:%d", ...).c_str());
```

Reply:
I'm also wondering now is http+unix:// a better prefix for the unix socket case

Reply:
it will look weird for most users. trust me, more than 90% users out there don't even know what is a unix socket